src/main/java/org/minions/devfund/yuri/movieDataBase/Actor.java

```diff
@@ -0,0 +1,77 @@
+package org.minions.devfund.yuri.movieDataBase;
+
+import java.util.ArrayList;
+
+/**
+ * Represents an Actor with all attributes and behavior.
+ */
+public class Actor {
+    private String name;
+    private ArrayList<Movie> movies;
+
+    /**
+     * Constructor method.
+     *
+     * @param name actor name.
+     */
+    public Actor(final String name) {
+        this.name = name;
+        this.movies = new ArrayList<>();
+    }
+
+    /**
+     * Constructor method by default.
+     */
+    public Actor() {
+        this("");
+    }
+
+    /**
+     * Sets the name of the actor.
+     *
+     * @param name new actor name.
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Sets the movies related with the actor.
+     *
+     * @param movies list of movies.
+     */
+    public void setMovies(final ArrayList<Movie> movies) {
+        this.movies = movies;
+    }
+
+    /**
+     * Gets the actor name.
+     *
+     * @return actor name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Gets the actor movie list.
+     *
+     * @return the movie list.
+     */
+    public ArrayList<Movie> getMovies() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)